packages/edit-site/src/components/global-styles/font-library-modal/installed-fonts.js

```diff
@@ -93,7 +93,11 @@ function InstalledFonts() {
 			const { canUser } = select( coreStore );
 			return (
 				customFontFamilyId &&
-				canUser( 'delete', 'font-families', customFontFamilyId )
+				canUser( 'delete', {
+					kind: 'postType',
+					name: 'wp_font_family',
+					id: customFontFamilyId,
```

Comment:
I appreciate the explicit syntax, it makes it clearer as to what we're doing. Previously, one had to search what `font-families` is - is it a taxonomy, is it a post type?